docker-compose.yml

```diff
@@ -216,6 +216,22 @@ services:
     ports:
       - 7000:8080
 
+  pdf-service-api:
+    image: node:14-alpine
+    environment:
+      GOTENBERG_URL: http://gotenberg:4000
+      LOGGER_LEVEL: "debug"
+      SERVER_SHOW_ERRORS: "true"
+    ports:
+      - 3003:3000
+    working_dir: /opt/app
+    links:
+      - appeals-service-api
```

Comment:
Does this need a connection to the `appeals-service-api`? If so, we'll need an envvar for it.

Also, this will need `gotenberg` adding else it won't be able to connect via the internal DNS

Reply:
The pdf service does not need to call out to the appeal api service. i have replaced that line with '- gotenberg' as you suggested but everything seemed to be working ok in local docker previously.